client/state/analytics/test/middleware.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
+
+import {
+	withAnalytics,
+	bumpStat,
+	recordGoogleEvent,
+	recordGooglePageView,
+	recordTracksEvent,
+	recordPageView
+} from '../actions';
+import analyticsMock from './analytics-mock';
+
+describe( 'Analytics Middleware', () => {
```

Comment:
Conventionally, we'd omit the "Analytics" prefix here, as it's generated by our test runner based on the JSON structure of `client/tests.json`, i.e. `middleware` alone would still be output as:

```
state
	analytics
		middleware
			✓ should do foo
			✓ should do bar
```

... in the runner.